plugins/flytekit-envd/tests/test_image_spec.py

```diff
@@ -101,7 +101,7 @@ def build():
 
 def test_image_spec_extra_index_url():
     image_spec = ImageSpec(
-        packages=["-U --pre pandas", "torch", "torchvision"],
```

Comment:
Remove `--pre` here since envd will install pandas 3.0.0.dev0 and cause an error.